clouddq/integration/bigquery/bigquery_client.py

```diff
@@ -208,3 +208,59 @@ def execute_query(
         )
 
         return query_job
+
+    def get_table_schema(self, table: str, project_id: str = None) -> dict:
```

Comment:
Can we please add an integration test for this behaviour?

Reply:
added the tests for this behaviour